Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -769,6 +759,20 @@ public void SetLogLevel(LogCategory Category, LogLevel Level)
                 }
             }
 
+            //-------------------------------------------------------------------------
+            /// <summary>
+            /// Initialize log levels loaded from <see cref="LogLevelConfig" />
```

Comment:
+100 for using cool `cref` comment linkage. Makes me want to up my own commenting game 😃